src/main/resources/migrations/00019_alter_transactions_requests_mandate_id.sql

```diff
@@ -0,0 +1,35 @@
+--liquibase formatted sql
+
+--changeset uk.gov.pay:alter_table-transactions-mandate-id
+ALTER TABLE transactions ADD COLUMN mandate_id BIGINT;
+--rollback ALTER TABLE transactions DROP COLUMN mandate_id;
+
+--changeset uk.gov.pay:add_transactions_mandates_fk
+ALTER TABLE transactions ADD CONSTRAINT transactions_mandates_fk FOREIGN KEY (mandate_id) REFERENCES mandates (id);
+--rollback drop constraint transactions_mandates_fk;
+
+--changeset uk.gov.pay:alter_table-payers-payment-requests-fk
+ALTER TABLE transactions DROP CONSTRAINT transactions_payment_requests_fk
+--rollback add constraint transactions_payment_requests_fk FOREIGN KEY (payment_request_id) REFERENCES payment_requests (id);
+
+--changeset uk.gov.pay:alter_table-transactions-mandate-id
+ALTER TABLE transactions ADD COLUMN mandate_id BIGINT;
+--rollback ALTER TABLE transactions DROP COLUMN mandate_id;
+
+--changeset uk.gov.pay:alter_table-transactions-external_id
+ALTER TABLE transactions ADD COLUMN external_id VARCHAR(30);
```

Comment:
Can `external_id` be `VARCHAR(26)` if you are using https://github.com/alphagov/pay-direct-debit-connector/blob/b39cef769e099ece3ff974c585375c4aa85688b9/src/main/java/uk/gov/pay/directdebit/common/util/RandomIdGenerator.java#L21 ?

Reply:
Yes, I will make sure mandate and transaction external id are both VARCHAR(26)